src/addons/addons/mediarecorder/style.css

```
@@ -55,45 +55,72 @@ p.mediaRecorderPopupOption {
+  --mediaRecorderAccent: var(--motion-primary, var(--loop-accent, var(--editorDarkMode-accent, var(--looks-secondary, #4c97ff))));
+ 
+  background: var(--mediaRecorderAccent);
+  background-image: linear-gradient(
+    to bottom,
+    rgba(255, 255, 255, 0.3) 0%,
+    rgba(255, 255, 255, 0.05) 50%,
+    rgba(0, 0, 0, 0.05) 51%,
+    rgba(0, 0, 0, 0) 100%
+  );
+  
+  box-shadow: inset 0 1px 0 rgba(255, 255, 255, 0.4);
```

Stylelint (17.4.0)

[error] Unexpected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)